### ISSUE TITLE
Monitor updates in the Site::setMinPasswordLength

### ISSUE THREAD

**Issue by jonasraoni** [ulsdevteam/pkp-betterPassword#18]
Since the cache is discarding smaller passwords, we need to monitor changes in this setting (or inflate the cache with small passwords), that could happen outside the plugin.

Still talking about the cache... If we sort the hashes before storing them into the disk, we could use a binary-search instead of using the `opcache`. But I think it's not worth, unless the file is huge :D

**Comment by ctgraham** [comment]
I think the cache should be dependent on the Site's `minPasswordLength` setting.  If the setting changes, the cache is invalid and needs to be recreated.

**Comment by jonasraoni** [comment]
Yep, I've added a local [check here](https://github.com/ulsdevteam/pkp-betterPassword/commit/62116c957fb046d1c241ba4b79fed23c58f34db4#diff-9665e33fe4ac87750ad8028e687a1d686e03f1d194e83f6eb113a977b7b9ea8cR166), but I think it should be done through a hook.
